core/lexer.cpp

```diff
@@ -71,32 +70,29 @@ static void lex_ws(const char *&c, unsigned &new_lines, unsigned &indent, const
     for (; *c != '\0' && (*c == ' ' || *c == '\n' || *c == '\t' || *c == '\r'); c++) {
         switch (*c) {
             case '\r':
-            // Ignore.
-            break;
+                // Ignore.
+                break;
 
             case '\n':
-            indent = 0;
-            new_lines++;
-            line_number++;
-            line_start = c + 1;
-            break;
+                indent = 0;
+                new_lines++;
+                line_number++;
+                line_start = c + 1;
+                break;
 
             case ' ':
-            indent += 1;
-            break;
+                indent += 1;
+                break;
 
             // This only works for \t at the beginning of lines, but we strip it everywhere else
             // anyway.  The only case where this will cause a problem is spaces followed by \t
             // at the beginning of a line.  However that is rare, ill-advised, and if re-indentation
             // is enabled it will be fixed later.
-            case '\t':
-            indent += 8;
-            break;
+            case '\t': indent += 8; break;
```

Comment:
`clang-format` can sometimes produce weird results. Some quick experimentation shows that the reason why the `' '` case is not put in one line is the comment below it. But I have no idea why.